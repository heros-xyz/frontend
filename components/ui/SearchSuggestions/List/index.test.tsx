import { render } from "@testing-library/react";
import mockRouter from "next-router-mock";
import SearchSuggestionsList from ".";
jest.mock("next/router", () => require("next-router-mock"));

it("renders List unchanged", () => {
  mockRouter.push("/initial-path");
  const onShowAllResult = () => {};
  const { container } = render(
    <SearchSuggestionsList
      onClick={() => {}}
      buttonName="See all results"
      items={[
        {
          id: "123",
          avatar: "",
          fullName: "Matt Prior",
          sport: "Cricket",
          sourceSubscriptionsTotal: 10,
        },
        {
          id: "123",
          avatar: "",
          fullName: "Mavis Breitenberg",
          sport: "Marathon",
          sourceSubscriptionsTotal: 10,
        },
        {
          id: "123",
          avatar: "",
          fullName: "Marcus Mariota",
          sport: "Football",
          sourceSubscriptionsTotal: 10,
        },
        {
          id: "123",
          avatar: "",
          fullName: "Marshall Eriksen",
          sport: "Surfing",
          sourceSubscriptionsTotal: 10,
        },
      ]}
      onShowAllResult={onShowAllResult}
    />
  );
  expect(container).toMatchSnapshot();
});
