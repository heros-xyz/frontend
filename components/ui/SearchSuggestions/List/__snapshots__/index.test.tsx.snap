// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders List unchanged 1`] = `
<div>
  <div
    class="css-p3diwk"
  >
    <div
      class="css-kcc1xl"
    >
      <div
        class="css-k008qs"
      >
        <div
          class="css-haoc2a"
        >
          <img
            alt="heros item"
            class="chakra-image css-k6jbwm"
            src=""
          />
        </div>
        <div
          class="css-zupw4a"
        >
          <p
            class="chakra-text css-wwubfu"
          >
            Matt Prior
          </p>
          <p
            class="chakra-text css-1sl60kx"
          >
            Cricket
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-kcc1xl"
    >
      <div
        class="css-k008qs"
      >
        <div
          class="css-haoc2a"
        >
          <img
            alt="heros item"
            class="chakra-image css-k6jbwm"
            src=""
          />
        </div>
        <div
          class="css-zupw4a"
        >
          <p
            class="chakra-text css-wwubfu"
          >
            Mavis Breitenberg
          </p>
          <p
            class="chakra-text css-1sl60kx"
          >
            Marathon
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-kcc1xl"
    >
      <div
        class="css-k008qs"
      >
        <div
          class="css-haoc2a"
        >
          <img
            alt="heros item"
            class="chakra-image css-k6jbwm"
            src=""
          />
        </div>
        <div
          class="css-zupw4a"
        >
          <p
            class="chakra-text css-wwubfu"
          >
            Marcus Mariota
          </p>
          <p
            class="chakra-text css-1sl60kx"
          >
            Football
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-kcc1xl"
    >
      <div
        class="css-k008qs"
      >
        <div
          class="css-haoc2a"
        >
          <img
            alt="heros item"
            class="chakra-image css-k6jbwm"
            src=""
          />
        </div>
        <div
          class="css-zupw4a"
        >
          <p
            class="chakra-text css-wwubfu"
          >
            Marshall Eriksen
          </p>
          <p
            class="chakra-text css-1sl60kx"
          >
            Surfing
          </p>
        </div>
      </div>
    </div>
    <div
      class="css-r9do2v"
    >
      <div
        class="css-1u71de6"
      >
        See all results
      </div>
    </div>
  </div>
</div>
`;
